modules/gapi/include/opencv2/gapi/gcompiled.hpp

```diff
@@ -208,6 +210,9 @@ class GAPI_EXPORTS GCompiled
     // FIXME: Why it requires compile args?
     void reshape(const GMetaArgs& inMetas, const GCompileArgs& args);
 
+    static void setMetaData(ade::Graph &g, cv::GCompileArgs &args
+                        , const cv::GMetaArgs &metas);
```

Comment:
Please fix formatting. Also it is not only about _set_ting, it also triggers a complex recursive process on inferring metadata in the whole graph. So I'd rename it to `inferMeta`